pkg/lib/bundle/generate.go

```diff
@@ -123,6 +126,33 @@ func GenerateFunc(directory, outputDir, packageName, channels, channelDefault st
 		return err
 	}
 
+	if synthesizeCSV {
+		if err := synthesize.Generate(directory, outputDir, overwrite); err != nil {
+			return err
+		}
+	}
+
+	if validateCSV {
```

Comment:
When do we _not_ want to validate the CSV?

Reply:
Good call. If the CSV is not synthesized, do we still validate it anyway?

Reply:
Based on the private discussion, changing the flag to `skip-validation`. Although this would cause defaulting validation and prints errors and warnings. If the bundle does not have a CSV, command error out. @ecordell